openapi.yml

```
@@ -3512,6 +3512,16 @@ paths:
           in: query
           name: created_after
           description: Restricts results to only those that were created after the provided date
+        - schema:
+            type: boolean
+          in: query
+          name: needs_confirmation
+          description: "Restricts results to only those that do not have an associated invite (`needs_confirmation: true`) or do have an associated invite (`needs_confirmation: false`)."
```

🌵 `only` is redundant imo.
What about explaining what confirmed means?

```suggestion
          description: "A confirmed `Registration` is one with an associated `Invite`. This filter returns those without an `Invite` when true, and those with an `Invite` when false."
```

---

This is better.